build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:0.12.2'
+        classpath 'com.android.tools.build:gradle:1.0.0'
```

Comment:
This is because we're updating two different things:
The Android-gradle plugin is updated from 0.12.2 to 1.0.0.
Gradle is updated to 2.2.1, because this is a dependency of the android-gradle v1.0.-